<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPreferencesDialog</class>
 <widget class="QDialog" name="GlobalPreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Global Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="preferencesSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="preferenceCategoryList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/wrench.png</normaloff>:/icons/wrench.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Display</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/monitor_edit.png</normaloff>:/icons/monitor_edit.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>File Paths</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/folders.png</normaloff>:/icons/folders.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Performance</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/processor.png</normaloff>:/icons/processor.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Language</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons.qrc">
         <normaloff>:/icons/locate.png</normaloff>:/icons/locate.png</iconset>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="preferencePageStack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="generalPreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="generalTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>General</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="generalTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QGroupBox" name="startupOptionsGroup">
              <property name="title">
               <string>Start Up</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_27">
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>Geometrize application logo on launch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="text">
                  <string>Load recent files list on launch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_4">
                 <property name="text">
                  <string>Load templates and samples on launch</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="miscOptionsGroup">
              <property name="title">
               <string>Misc</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_26">
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="text">
                  <string>Auto-start geometrizing when image job is opened</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="displayPreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="displayTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="displayTitleSpacer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QGroupBox" name="launchPageDisplayOptionsGroup">
              <property name="title">
               <string>Launch Page Display</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QCheckBox" name="showLaunchWindowConsoleByDefault">
                   <property name="text">
                    <string>Show Launch Console By Default</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="imageJobDisplayOptionsGroup">
              <property name="title">
               <string>Image Geometrize Display</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QCheckBox" name="showImageJobConsoleByDefault">
                   <property name="text">
                    <string>Show Image Geometrizing Console By Default</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="showScriptEditorsByDefault">
                   <property name="text">
                    <string>Show Script Editors By Default</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_5">
                   <property name="text">
                    <string>Show Pixmap Results View By Default</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_6">
                   <property name="text">
                    <string>Show Vector Results View By Default</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filePathPreferencesPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="filePathsTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>File Paths</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="filePathsTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QGroupBox" name="defaultLoadPathOptionsGroup">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Default Load Paths</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_23">
               <item>
                <widget class="geometrize::dialog::FolderSelectionWidget" name="widget_2" native="true"/>
               </item>
               <item>
                <widget class="geometrize::dialog::FolderSelectionWidget" name="widget_4" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="defaultSavePathOptionsGroup">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Default Save Paths</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_24">
               <item>
                <widget class="geometrize::dialog::FolderSelectionWidget" name="widget_3" native="true"/>
               </item>
               <item>
                <widget class="geometrize::dialog::FolderSelectionWidget" name="widget_5" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="templatePathOptionsGroup">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Template Paths</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_25">
               <item>
                <widget class="geometrize::dialog::FolderSelectionWidget" name="widget_6" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="performancePreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="performanceTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Performance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="performanceTitleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QGroupBox" name="imageResizingGroup">
              <property name="title">
               <string>Image Downscaling</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QCheckBox" name="imageResizeCheckbox">
                   <property name="text">
                    <string>Enabled</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="resizeThresholdLabel">
                   <property name="toolTip">
                    <string>Images larger than this will be scaled down</string>
                   </property>
                   <property name="text">
                    <string>Resize Threshold</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_19">
                   <item>
                    <layout class="QFormLayout" name="formLayout_2">
                     <property name="formAlignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="resizeWidthLabel">
                       <property name="text">
                        <string>Width</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QSpinBox" name="resizeWidthSpinBox">
                       <property name="maximum">
                        <number>65536</number>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="resizeHeightLabel">
                       <property name="text">
                        <string>Height</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QSpinBox" name="resizeHeightSpinBox">
                       <property name="maximum">
                        <number>65536</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="threadingOptionsGroup">
              <property name="title">
               <string>Multithreading</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="threadsPerImageJobLabel">
                     <property name="text">
                      <string>Max Threads Per Image Job</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="maxThreadsPerImageJobSpinBox">
                     <property name="maximum">
                      <number>128</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="languagePreferencesPage">
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="languageTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="languageTitleSpacer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="geometrize::dialog::LanguageSelectWidget" name="widget" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="closeButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>geometrize::dialog::LanguageSelectWidget</class>
   <extends>QWidget</extends>
   <header>dialog/languageselectwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::FolderSelectionWidget</class>
   <extends>QWidget</extends>
   <header>dialog/folderselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
